Sources/Action.swift

```diff
@@ -84,7 +84,14 @@ public final class Action<Input, Output, Error: Swift.Error> {
 	///                given the latest `Action` state.
 	///   - execute: A closure that produces a unit of work, as `SignalProducer`, to be
 	///              executed by the `Action`.
-	public init<State: PropertyProtocol>(state property: State, enabledIf isEnabled: @escaping (State.Value) -> Bool, execute: @escaping (State.Value, Input) -> SignalProducer<Output, Error>) {
+	public convenience init<State: PropertyProtocol>(state property: State, enabledIf isEnabled: @escaping (State.Value) -> Bool,
+										 execute: @escaping (State.Value, Input) -> SignalProducer<Output, Error>) {
+		self.init(state: property, enabledIf: isEnabled, isExecuting: { $0.isExecuting }, execute: execute)
+	}
+
+	fileprivate init<State: PropertyProtocol>(state property: State, enabledIf isEnabled: @escaping (State.Value) -> Bool,
+											 isExecuting: @escaping (ActionState) -> Bool,
```

Comment:
The only piece of internal structure that I *had* to expose is hooking into `Action.isExecuting`. I think a tidier abstraction would expose more though.